e2e/each/__tests__/failure.test.js

```diff
@@ -49,9 +49,9 @@ test.each(['red', 'green', 'bean'])(
 );
 
 describe.each`
-  left   | right
-  ${'a'} | ${'b'}
-  ${'c'} | ${'d'}
+  left     | right
+  ${'"a"'} | ${'"b"'}
+  ${'"c"'} | ${'"d"'}
```

Comment:
I'd add the quotes in the message below, instead of in the string value.

Reply:
+1